fluent/src/resolver.js

```diff
@@ -14,24 +15,14 @@
  * information.
  *
  * In case of errors the resolver will try to salvage as much of the
- * translation as possible.  In rare situations where the resolver didn't know
+ * translation as possible. In rare situations where the resolver didn't know
  * how to recover from an error it will return an instance of `FluentNone`.
  *
  * All expressions resolve to an instance of `FluentType`. The caller should
  * use the `toString` method to convert the instance to a native value.
  *
- * All functions in this file pass around a special object called `scope`.
- * This object stores a set of elements used by all resolve functions:
- *
- *  * {FluentBundle} bundle
- *      bundle for which the given resolution is happening
- *  * {Object} args
- *      list of developer provided arguments that can be used
- *  * {Array} errors
- *      list of errors collected while resolving
- *  * {WeakSet} dirty
- *      Set of patterns already encountered during this resolution.
- *      This is used to prevent cyclic resolutions.
+ * Functions in this file pass around an instance of the `Scope` class, which
+ * stores the data required for successful resolution and error recovery.
```

Comment:
Side question, is this rendered in the esdocs somewhere? I couldn't find it, and I wonder if that's a bug or a feature.

Also, can't comment on that inline, we don't do list formats or more right now?

Should we also tweak `bundle's language` to `languages`, as it can have more than one?

Reply:
> Side question, is this rendered in the esdocs somewhere? I couldn't find it, and I wonder if that's a bug or a feature.

I think it's a missing featre. The `@overview` tag is [supported in JSDoc](https://jsdoc.app/tags-file.html). I don't see it in the list of [tags supported by ESDoc](https://esdoc.org/manual/tags.html), though.

> Also, can't comment on that inline, we don't do list formats or more right now?

Correct. Fixed.

> Should we also tweak bundle's language to languages, as it can have more than one?

Fixed.

Reply:
Found https://github.com/esdoc/esdoc/issues/66, it's wontfix.